ledger/ledger-api-tests/suites/BUILD.bazel

```diff
@@ -41,6 +41,10 @@ versions = {
     da_scala_library(
         name = "suites-%s" % lf_version,
         srcs = config["entrypoints"] + config["srcs"],
+        scala_deps = [
+            "@maven//:org_scalatest_scalatest_core",
```

Comment:
This looks odd. In the final implementation, I'd decouple the tagging annotations from the Scalatest lib, at least for the scope of the ledger-api-test-tool. Fine for the PoC scope, though

Reply:
I've added this concern to the design doc. The lib part of the approach would not have scalatest dependency and only contain tags. (In the form of annotations or case classes, to decide later)